### ISSUE TITLE
Support for parameter file schemas

### ISSUE THREAD

**Issue by BernieWhite** [microsoft/PSRule.Rules.Azure#674]
**Description of the issue**

When parameter files with alternative schemas such as `https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentParameters.json#` are used with `Get-AzRuleTemplateLink` an error is reported suggested that a metadata property is missing.

**Expected behaviour**

1. Alternative schemas should be processed.
2. The error message that a metadata property is missing is incorrect.

**Error output**

```text
Get-AzRuleTemplateLink: The parameter file '<removed>' does not contain a metadata property.
```

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **1.1.0**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.1.2
PSEdition                      Core
GitCommitId                    7.1.2
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Additional context**

This was previously an issue with older versions < 1.1.0 however an error message was not reported, the parameter file was skipped.
